config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -341,7 +341,7 @@ presubmits:
       nodeSelector:
         dedicated: "ubuntu"
       containers:
-      - image: gcr.io/k8s-testimages/krte:v20191017-d96ae13-master
+      - image: gcr.io/k8s-testimages/krte:latest-master
```

Comment:
don't do this. we won't pull it every time so we'll have an arbitrarily stale version.

the daily test-infra bump PRs will automatically update the v images

Reply:
ahh, ok, didn't know there was such thing
/close

Reply:
you can always bump ahead earlier, if you visit the image as a url you can see what the tag resolves to for latest, or you can check the postsubmits for a version

using `latest-foo` as a tag without `imagePullPolicy` is a bug, kubernetes only magically defaults pull policy if the tag is exactly `latest`